lib/users.rb

```diff
@@ -0,0 +1,21 @@
+require 'pg'
+
+class Users
+
+  def self.all
+    @peeps = []
+    if ENV['ENVIRONMENT'] = 'test'
+      connection = PG.connect(dbname: 'chitter_test' )
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    result = connection.exec("SELECT * FROM messages ORDER BY time DESC")
+    # result.map do |messages| messages['content']
```

Comment:
Nice, clean, easy
I also like how you've left your commented out prior versions to see where you've built from, as it's still in development